### PR TITLE
fix(app, tabs): make page work in IE 11 and fix Tabs titles

### LINKED ISSUE #14: Tabs titles re-render doesn't work in IE, FF
- this is probably because non native browsers doesn't trigger property changes in slotted elements, so no slotchange is triggered
skate-tab-titles-bug

### PR DESCRIPTION
**Pull request by Hotell**
Closes #14 #15

### CODE REVIEW

**Comment by treshugart** [comment]
Also closes https://github.com/skatejs/skatejs/issues/1025.

**Review by treshugart — APPROVED** [review]

File: src/app/index.js [hunk 1]
```diff
@@ -9,7 +9,8 @@ import Module from '../module';
 import Route from '../route';
 import title from '../_/title';
 
-export default define('sk-app', class extends Component {
+export default define(class extends Component {
+  static get is(){ return 'sk-app' }
```
Comment (treshugart) [inline src/app/index.js:13]: nit: Preferred code style: 

```js
static get is () {
  return 'sk-app';
}
```

Or simply:

```js
static is = 'sk-app';
```

I generally do the latter.

File: src/module.js [hunk 2]
```diff
@@ -1,20 +1,23 @@
...
+      this.load(args => (this.args = args));
     }
-    return elem.args;
-  },
-  render (elem) {
-    if (typeof elem.done === 'function') {
-      return elem.done(elem.args);
+    return this.args;
+  }
+  renderCallback () {
```
Comment (treshugart) [inline src/module.js:18]: You can also destructure `done` as an argument:

```js
renderCallback ({ done })
```

**Comment by treshugart** [comment]
Thanks for fixing this! Approved, just had a few minor nits that you may want to look at before merging.

File: src/app/index.js (hunk 1 above)
Comment (Hotell) [reply src/app/index.js:13]: yeah I to the former all the time :D as we had discussion previously getter is immutable, property not.

File: src/module.js (hunk 2 above)
Comment (Hotell) [reply src/module.js:18]: yeah I know, just wanted to remove those warnings from console log

**Comment by Hotell** [comment]
sure thing! my pleasure. It's great to have it work cross browser ( for instance when spreading skate love at ngParty meetup. Now we have solid reference what is possible and that it works everywhere )

**Comment by Hotell** [comment]
Maybe those suggestions should be addressed in next PR, if you really wanna tweak that, wdyt? thanks!

**Pull request merged**